serde/src/java/org/apache/hadoop/hive/serde2/json/HiveJsonReader.java

```diff
@@ -55,9 +56,7 @@
 import org.apache.hadoop.hive.serde2.objectinspector.StructField;
 import org.apache.hadoop.hive.serde2.objectinspector.StructObjectInspector;
 import org.apache.hadoop.hive.serde2.objectinspector.primitive.PrimitiveObjectInspectorFactory;
-import org.apache.hadoop.hive.serde2.typeinfo.BaseCharTypeInfo;
-import org.apache.hadoop.hive.serde2.typeinfo.PrimitiveTypeInfo;
-import org.apache.hadoop.hive.serde2.typeinfo.TimestampLocalTZTypeInfo;
+import org.apache.hadoop.hive.serde2.typeinfo.*;
```

Comment:
This is the last request hopefully. We don't use a wildcard import.
https://github.com/apache/hive/blob/da272b45f7707c41713dbfd50d61f0ff285d099f/checkstyle/checkstyle.xml#L155

Reply:
Fixed!